Blackjack/Game/BlackjackGame.m

```diff
@@ -0,0 +1,246 @@
+//
+//  BlackjackGame.m
+//  Blackjack
+//
+//  Created by Alex on 28/09/2022.
+//
+
+#import "BlackjackGame.h"
+#import "PlayingCard.h"
+
+// MARK: - Private Properties
+
+@interface BlackjackGame()
+@property (nonatomic, strong) Deck *deck;
+@property (nonatomic) NSInteger cardsAmountToWin;
+@property (nonatomic) NSInteger dealerMinimumCardEvaluation;
+@property (nonatomic) enum State gameState;
+@property (nonatomic) NSArray *gameStates;
+@end
+
+// MARK: - Implementation
+
+@implementation BlackjackGame
+
+@synthesize dealer = _dealer; // because we provide setter AND getter
+
+enum State {
+    collectBets, dealCards, awaitingPlayersDecision, awaitingDealerDecision
+};
+
+- (Contestant *) dealer {
+    if (!_dealer) _dealer = [[Contestant alloc] initWith:@"Dealer" cards:[[NSMutableArray alloc] init] chips: 0 isPlaying:YES];
+    return _dealer;
+}
+
+// MARK: - Static Methods
+
++ (NSDictionary<NSString*, NSNumber*> *)cardValues {
+    return @{@"2": @2, @"3": @3, @"4": @4, @"5": @5, @"6": @6, @"7": @7, @"8": @8, @"9": @9, @"10": @10, @"J": @10, @"Q": @10, @"K": @10};
+}
+
++ (NSInteger)cardsAmountToWin {
+    return 21;
+}
+
++ (NSInteger)dealerMinimumCardEvaluation {
+    return 16;
+}
+
+- (NSArray *)currentActivePlayers {
+    NSMutableArray *activePlayers = [[NSMutableArray alloc] init];
```

Comment:
You can use `new` instead of `alloc ] init]`

Reply:
Great to know :)